boards/arm/efm32gg_slwstk6121a/board.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 Interay Solutions B.V.
+ * Copyright (c) 2019 Oane Kingma
+ * Copyright (c) 2020 Thorvald Natvig
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
+#include <sys/printk.h>
+#include "em_cmu.h"
+#include "board.h"
+
+static int efm32gg_slwstk6121a_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+#ifdef CONFIG_ETH_GECKO
```

Comment:
Is this ifdef necessary? It looks like the CMake file does not process this file as a source `ifndef CONFIG_ETH_GECKO`

Reply:
It's not needed yet. It will be needed when the WF200 driver is pushed, but that will also need changes to CMake, so right now it is inconsistent.
Removed.